integrationtests/src/main/java/io/kroxylicious/proxy/testkafkacluster/KafkaClusterConfig.java

```diff
@@ -154,8 +169,26 @@ public Stream<ConfigHolder> getBrokerConfigs(Supplier<KafkaEndpoints> endPointCo
     protected Map<String, Object> getConnectConfigForCluster(String bootstrapServers) {
         Map<String, Object> kafkaConfig = new HashMap<>();
         String saslMechanism = getSaslMechanism();
+        String securityProtocol = getSecurityProtocol();
+
+        if (securityProtocol != null) {
+            kafkaConfig.put(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, securityProtocol);
+
+            if (securityProtocol.contains("SSL")) {
+                kafkaConfig.put("ssl.truststore.location", KeytoolCertificateGenerator.getCertLocation());
+                kafkaConfig.put("ssl.truststore.password", KeytoolCertificateGenerator.getPassword());
+                if (securityProtocol.equals(SecurityProtocol.SSL.toString())) {
```

Comment:
The version of this in `getBrokerConfigs` doesn't guard the keystore config with a guard to ensure the protocol is `SSL`. Should the guard be included in the `getBrokerConfigs` version or relaxed here?

Reply:
This is done because the configuration is different for the client if we use `SSL` or `SASL_SSL`. Both security protocols uses the `ssl.truststore.*` parameters, but the `ssl.keystore.*` are only needed for `SSL`.